server/db/index.js

```diff
@@ -5,32 +5,38 @@ const db = require("./db");
 const User = require("./models/User");
 const Product = require("./models/Product");
 const Category = require("./models/Category");
-
 const Order = require("./models/Order");
-
 const LineItem = require("./models/Lineitem");
 
-
 //TODO: associations could go here!
 //Product - Category = Many to Many
 
 Product.belongsToMany(Category, { through: "product_categories" });
 Category.belongsToMany(Product, { through: "product_categories" });
 
-//TODO: Lineitem  - Order should have associations
-// LineItem.belongsTo(Order)
-// Order.hasMany(LineItem)
+//Product - Order
+// Product.belongsTo(Order);
+// Order.hasMany(Product);
+
+//Lineitem  - Order
+LineItem.belongsTo(Order);
```

Comment:
These associations would probably be better served as a many-many relationship between Orders and Products _through_ the LineItem model

something like `Order.hasMany(Product, { through: LineItem })`

This way you could do things like `await order.getProducts()` to get all product information attached to the order